web/third/bootstrap/css/bootstrap.portico.css

```diff
@@ -243,7 +243,7 @@ input:focus:invalid:focus {
   display: none;
 }
 .form-inline label,
-.form-inline .btn-group {
+.form-inline .bootstrap-btn-group {
   display: inline-block;
```

Comment:
I think this style is unused? So we could just delete this line of CSS instead of renaming it.